Changelog.md

```diff
@@ -1,5 +1,12 @@
 # Unreleased
 
+* [Remove "UEFI boot" log message](https://github.com/rust-osdev/bootloader/pull/476)
```

Comment:
We should add a new `# 0.11.10 – 2025-02-10` section heading and link the full change log below `**Full Changelog**: https://github.com/rust-osdev/bootloader/compare/v0.11.9...v0.11.10`

Reply:
Thanks, done!